cmd2/cmd2.py

```diff
@@ -502,23 +502,26 @@ def pexcept(self, msg: Any, *, end: str = '\n', apply_style: bool = True) -> Non
         # Set apply_style to False since style has already been applied
```

Comment:
Should `pwarning()` have an `apply_style` optional argument added to it to be consistent with `perror()` and `pexcept()`?

Reply:
See discussion on #815 for this question.